From 8db421228020b0e195add49aa8ed332c87d49cf2 Mon Sep 17 00:00:00 2001
From: Seán de Búrca <leftmostcat@gmail.com>
Date: Tue, 27 Aug 2013 15:07:07 +0000
Subject: Fix autogen with latest gnome-common and intltool failure in config

Side-by-side use of IT_PROG_INTLTOOL and AM_GNU_GETTEXT is not supported.

https://bugzilla.gnome.org/show_bug.cgi?id=705378
---
diff --git a/Makefile.am b/Makefile.am
index 1ea7333..1521128 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -6,6 +6,8 @@ if ENABLE_DOCUMENTATION
 SUBDIRS += help
 endif
 
+ACLOCAL_AMFLAGS = -I m4 ${ACLOCAL_FLAGS}
+
 EXTRA_DIST = intltool-extract.in intltool-update.in intltool-merge.in ChangeLog.pre-2.27 gtk-doc.make
 
 DISTCHECK_CONFIGURE_FLAGS = --disable-schemas-install \
diff --git a/configure.ac b/configure.ac
index 9305910..fe61daa 100644
--- a/configure.ac
+++ b/configure.ac
@@ -8,6 +8,7 @@ AC_INIT([gnome-bluetooth],
         [gnomebt_version_major.gnomebt_version_minor.gnomebt_version_micro],
         [http://bugzilla.gnome.org/enter_bug.cgi?product=gnome-bluetooth])
 
+AC_CONFIG_MACRO_DIR([m4])
 AC_CONFIG_SRCDIR([applet/bluetooth-applet.c])
 AC_CONFIG_HEADERS([config.h])
 
@@ -41,9 +42,6 @@ AM_PROG_LIBTOOL
 # i18n stuff
 IT_PROG_INTLTOOL([0.40.0])
 
-AM_GNU_GETTEXT([external])
-AM_GNU_GETTEXT_VERSION([0.17])
-
 GETTEXT_PACKAGE=gnome-bluetooth2
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE",
--
cgit v0.9.2
